pkg/ccl/logictestccl/testdata/logic_test/crdb_internal

```diff
@@ -176,3 +176,37 @@ SELECT crdb_internal.stream_ingestion_stats_json(1);
 
 query error pq: crdb_internal\.stream_ingestion_stats_pb\(\): unimplemented
 SELECT crdb_internal.stream_ingestion_stats_pb(1);
+
+subtest show_tenant_capabilities_cache
+
+statement ok
+SELECT crdb_internal.create_tenant(5)
+
+query ITT colnames,retry
+SELECT (row).tenant_id, (row).capability_name, (row).capability_value
+FROM [SELECT crdb_internal.show_tenant_capabilities_cache() AS row]
```

Comment:
Can we simplify this to something like the following?

```suggestion
query ITT colnames,retry
SELECT tenant_id, capability_name, capability_value
FROM crdb_internal.show_tenant_capabilities_cache()
```

Reply:
Interesting, the syntax I suggested worked for me at the SQL shell on your build:

```
demo@127.0.0.1:26257/movr> select version();                                                                                                                                     
                                                       version
---------------------------------------------------------------------------------------------------------------------
  CockroachDB CCL v23.1.0-alpha.4-250-gd73d481857c (x86_64-apple-darwin22.3.0, built 2023/03/03 09:23:50, go1.19.4)
(1 row)

Time: 1ms total (execution 1ms / network 0ms)

demo@127.0.0.1:26257/movr> SELECT tenant_id, capability_name, capability_value                                                                                                   
                        -> FROM crdb_internal.show_tenant_capabilities_cache();                                                                                                  
  tenant_id |    capability_name    | capability_value
------------+-----------------------+-------------------
          1 | can_admin_split       | false
          1 | can_view_node_info    | false
          1 | can_view_tsdb_metrics | false
(3 rows)

Time: 1ms total (execution 0ms / network 0ms)
```

But, makes sense to me as a virtual table anyway.

Reply:
Weird I must have been doing something else wrong then since it was complaining about columns not existing.